test/conformance/source/environment.cpp

```diff
@@ -196,6 +287,26 @@ void uur::PlatformEnvironment::TearDown() {
 PlatformEnvironment::PlatformOptions
 PlatformEnvironment::parsePlatformOptions(int argc, char **argv) {
```

Comment:
This function isn't as big but has some dense sections that could maybe be split up too.

Reply:
I'm not sure. This lambda is only used in this one function, so IMO it doesn't really make sense to split it out into a free function.

Reply:
No problem, I just like to split functions up for readability once they get over a certain length